Tests/KernelTests/main.mm

```diff
@@ -586,8 +548,57 @@ - (void)testLargeBinary {
 
 #pragma mark - Main
 
+- (void)unloadDaemon {
+  NSTask *t = [[NSTask alloc] init];
+  t.launchPath = @"/bin/launchctl";
+  t.arguments = @[ @"unload", @"com.google.santad" ];
+  t.standardOutput = t.standardError = [NSPipe pipe];
+  [t launch];
+  [t waitUntilExit];
+}
+
+- (void)unloadExtension {
+  // Don't check the status of this, the kext may not be loaded..
+  OSStatus ret = KextManagerUnloadKextWithIdentifier(CFSTR("com.google.santa-driver"));
+  if (ret != kOSReturnSuccess && ret != kOSKextReturnNotFound) {
+    NSLog(@"Failed to unload extension: 0x%X", ret);
```

Comment:
0x%X --> %#X

Reply:
I don't like the way that outputs, 0x%X = 0xDC008014 while %#X = 0XDC008014.